### PR TITLE
Listing Translation Pt

### PR DESCRIPTION
**Pull request by dragonstyle**

### CODE REVIEW

File: src/resources/language/_language-pt.yml [hunk 1]
```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Equação"
 crossref-lof-title: "Lista de Figuras"
 crossref-lot-title: "Lista de Tabelas"
 crossref-lol-title: "Lista de Listagens"
+
+listing-page-order-by: "Ordenar por"
+listing-page-order-by-default: "Pré-selecionado"
+listing-page-order-by-date-asc: "O mais novo"
```
Comment (cscheid) [inline src/resources/language/_language-pt.yml:57]: Does `date-asc` show newest-date first? That's what the translation implies but it feels to me that "date ascending" would be oldest-date first.

File: src/resources/language/_language-pt.yml [hunk 2]
```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Equação"
...
+listing-page-order-by-date-desc: "Mais velho"
+listing-page-order-by-number-desc: "Descendente"
+listing-page-order-by-number-asc: "Ascendente"
```
Comment (cscheid) [inline src/resources/language/_language-pt.yml:60]:
```
listing-page-order-by-number-desc: "Decrescente"
listing-page-order-by-number-asc: "Crescente"
```

File: src/resources/language/_language-pt.yml [hunk 3]
```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Equação"
...
+listing-page-field-date: "Encontro"
```
Comment (cscheid) [inline src/resources/language/_language-pt.yml:61]:
```
listing-page-field-date: "Data"
```

("Encontro" translates to "will you go on a `date`" :)

Comment (dragonstyle) [reply src/resources/language/_language-pt.yml:61]: That wasn't quite what I had in mind!

File: src/resources/language/_language-pt.yml (hunk 1 above)
Comment (dragonstyle) [reply src/resources/language/_language-pt.yml:57]: No you're right the strings are reversed. Will fix!

**Pull request merged**